frontend/components/PhotoViewer.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import * as PropTypes from "prop-types";
 
-function getValue(dictionary, key, default_val) {
+export function getValue(dictionary, key, default_val) {
```

Comment:
It looks like this function is getting a key from a dictionary and, if undefined, returning a default value. This can actually be done using the or operator. For instance, instead of:

`x = getValue(dictionary, key, default_val)`

You can write:

`x = dictionary[key] || default_val`

See: https://stackoverflow.com/questions/9003999/how-to-get-value-from-object-with-default-value. No need to change all mentions of `getValue`, just a note for future reference.